helm-charts/powerdns/values.yaml

```diff
@@ -0,0 +1,150 @@
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+# volume mounts
+volumeMounts:
+  - name: pdns-additional-config-dir
+    mountPath: /etc/pdns/pdns.d
+
+  - name: pdns-recursor-config-dir
+    mountPath: /etc/pdns/pdns-recursor.d
+    
+  - name: pdns-persistent-storage
+    mountPath: /var/lib/pdns
+
+  - name: pdns-data-script
+    mountPath: /script/configure.sh
+    subPath: configure.sh
+
+  - name: pdns-init-db-script
+    mountPath: /script/init-db.sh
+    subPath: init-db.sh
+
+# volumes
+volumes:
+  - name: pdns-additional-config-dir
+    configMap:
+      name: pdns-additional-config-dir
+
+  - name: pdns-recursor-config-dir
+    configMap:
+      name: pdns-recursor-config-dir
+
+  - name: pdns-persistent-storage
+    persistentVolumeClaim:
+      claimName: powerdns
+
+  - name: pdns-data-script
+    configMap:
+      name: pdns-data-script
+      defaultMode: 0777
+
+  - name: pdns-init-db-script
+    configMap:
+      name: pdns-init-db-script
+      defaultMode: 0777
+
+# Environment Specific Values
+service:
+  type: ClusterIP
+  # Please provide a static ClusterIP from the range as shown by the following commands
+  # kubectl cluster-info dump | grep -m 1 service-cluster-ip-range
+  # OR
+  # SVCRANGE=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
+  # echo $SVCRANGE
+  #
+  # If the static IP is not in the range or is already used by another service, deployment will fail
+  # The reason we are using static IP is to avoid redeploying external-dns every time pdns is deployed.
+  ip: 10.43.0.11
+
+zone:
+  create: true
+  name: suse.baremetal
+  ns: ns1
+
+powerdns:
+  webserver:
+    address: 0.0.0.0
+    port: 8081
+    password: foo
+    allowfrom: 0.0.0.0/0,::/0
+  api:
+    key: abcdef123456
+  config: { dnsupdate: yes, loglevel: 6, default-soa-content: "ns1.suse.baremetal. admin.suse.baremetal. 0 10800 3600 604800 3600" }
+  recursor:
+    #To resolve internal addresses, you can add to the forward-zones, example: forward-zones: "suse.baremetal=127.0.0.1:54,example.com=100.100.100.100"
+    config: { forward-zones: "suse.baremetal=127.0.0.1:54",
+              local-address: 0.0.0.0:53,
+              dnssec: process-no-validate,
+              allow-from: "0.0.0.0/0,::/0",
+              max-cache-ttl: 600,
```

Comment:
I think @rtamalin  suggested adding some context or a pointer to the docs for the ttl settings

Reply:
Links to powerdns official documentation included.